packages/suite/src/components/suite/modals/Pin.tsx

```diff
@@ -41,13 +41,13 @@ export const Pin = ({ device, ...rest }: PinProps) => {
 
     return (
         <StyledModal
-            heading={
+            heading={<Translation id="TR_ENTER_PIN_ON_DEVICE_LABEL" />}
```

Comment:
Ok, also deleted `TR_ENTER_PIN_ON_DEVICE_LABEL` because it's no longer needed.

Also when I delete some translation key, how is that handled in Crowdin? Didn't find a button for proposing deletion

Reply:
will be deleted by next sync 👌